### PR TITLE
#1 the main functionality

### PR DESCRIPTION
**Pull request by Shimanoffsky**
add a comment

### CODE REVIEW

File: src/main/java/com/calculator/calculator/CalculatorApp.java [hunk 1]
```diff
@@ -18,6 +18,7 @@ public class CalculatorApp extends Application {
 
     public static void main(String[] args) {
         launch(args);
+        // comment
```
Comment (ivanserg0692) [inline src/main/java/com/calculator/calculator/CalculatorApp.java]: надо добавить плюс 1

Comment (Shimanoffsky) [reply src/main/java/com/calculator/calculator/CalculatorApp.java]: corrected

Comment (ivanserg0692) [reply src/main/java/com/calculator/calculator/CalculatorApp.java]: Хорошо

File: src/main/java/com/calculator/calculator/CalculatorApp.java [hunk 2]
```diff
@@ -18,6 +18,9 @@ public class CalculatorApp extends Application {
 
     public static void main(String[] args) {
         launch(args);
+        //ivan comment
+        // comment + 12
+        //next OK
```
Comment (Shimanoffsky) [inline src/main/java/com/calculator/calculator/CalculatorApp.java:23]: Ivan can you do something with this 12 please ?

Comment (ivanserg0692) [reply src/main/java/com/calculator/calculator/CalculatorApp.java:23]: Hello, I think everything is fine

**Pull request merged**